### PR TITLE
chore(operator-chart): Add etcd endpoint configuration value

### PR DESCRIPTION
**Pull request by sbckr**

### CODE REVIEW

File: klyshko-operator/charts/klyshko/values.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ controller:
     repository: carbynestack/klyshko-operator-controller
     pullPolicy: IfNotPresent
     tag: latest
+  etcdEndpoint: 172.18.1.129:2379
```
Comment (strieflin) [inline klyshko-operator/charts/klyshko/values.yaml:19]: This variable has to be added to the table in the chart README.

File: klyshko-operator/charts/klyshko/templates/deployment.yaml
Comment (strieflin) [inline klyshko-operator/charts/klyshko/templates/deployment.yaml:1]: Please update the copyright year in the files touched.

**Review by strieflin — CHANGES_REQUESTED** [review]

File: klyshko-operator/charts/klyshko/values.yaml (hunk 1 above)
Comment (sbckr) [reply klyshko-operator/charts/klyshko/values.yaml:19]: done

File: klyshko-operator/charts/klyshko/templates/deployment.yaml
Comment (sbckr) [reply klyshko-operator/charts/klyshko/templates/deployment.yaml:1]: done

**Review by strieflin — APPROVED** [review]

**Pull request merged**